### ISSUE TITLE
Docs unclear

### ISSUE THREAD

**Issue by callumacrae** [userapp-io/passport-userapp#3]
https://github.com/userapp-io/passport-userapp

The first code example is using Sequelize (I think?), but doesn't actually say so.

**Comment by timothyej** [comment]
Yes, that might be true. I've actually just copied the docs from passport's own strategies, but I will see if I can make it a little bit more clear. Thanks!

**Comment by callumacrae** [comment]
I'm now using just passport (ua doesn't really offer anything useful for this project), and the passport docs do the same: http://passportjs.org/guide/configure/
